### DISCUSSION TITLE
Poppler version is too high to compile

### DISCUSSION THREAD

**Discussion by Liujiahao0** [xournalpp/xournalpp#5115] [Q&A]
Before I installed Xournalpp, I had previously installed the source code version of poppler 23.06.0. However, after uninstalling poppler and then installing and compiling Xournalpp, it still prompted Package 'popper glib' requirements' poppler=0.86.1 'but version of poppler is 23.06.0.

**Comment by rolandlo** [comment]
All that is asked for in https://github.com/xournalpp/xournalpp/blob/e9ec033031db7a3a1cfca40af93353d161a01f3d/CMakeLists.txt#L100
is `poppler-glib >= 0.41.0`

I have poppler 22.10. It works with poppler 23.08 as well as shown on the flathub repo https://github.com/flathub/com.github.xournalpp.xournalpp/commit/e9ee2fbf65ad4a7b774b446fa3ab4a2fb29f3f32

Maybe you just have to delete the build folder and start over?! Where exactly did you get the error message?
